recipe/build.sh

```diff
@@ -1,10 +1,26 @@
 #!/usr/bin/env bash
 set -ex
 
-export FPM_C_COMPILER="${CC}"
-if [[ "${CONDA_BUILD_CROSS_COMPILATION:-0}" == 1 ]]; then
-  fpm install --compiler "${FC}" --flag "${LDFLAGS} ${FFLAGS}" --prefix "${PREFIX}"
+if [[ "${CONDA_BUILD_CROSS_COMPILATION:-0}" == "1" ]]; then
+  # MACOSX_DEPLOYMENT_TARGET is for the target_platform and not for build_platform
+  unset MACOSX_DEPLOYMENT_TARGET
```

Comment:
You need to set this back after the bootstrapper version has been built.

Reply:
Thanks @isuruf! 

@awvwgk, do you have time to send it as a new PR?